devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -203,7 +181,7 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
-    if (cmd.value == 0 && device.latestState("color").value != "#ffffff") {
+    if (device.latestState("color") == null || (cmd.value == 0 && device.latestState("color").value != "#ffffff")) {
         sendEvent(name: "color", value: "#ffffff", displayed: true)
```

Comment:
not sure why you are setting the color value to #ffffff every time off is received?